src/partials/languages/pl.html

```diff
@@ -39,8 +47,11 @@
   "meta.comments": "Komentarze",
   "meta.source": "Kod źródłowy",
   "nav": "Nawigacja",
+  "readtime.one": "Czas czytania: 1 min",
+  "readtime.other": "Czas czytania: # min",
+  "rss.created": "Kanał RSS",
+  "rss.updated": "Kanał RSS zaktualizowanych treści",
   "search": "Szukaj",
-  "search.config.pipeline": " ",
```

Comment:
Please don't remove this setting. This is exactly what I referred to in https://github.com/squidfunk/mkdocs-material/issues/4934#issuecomment-1408146570

Reply:
Oh, this is the template generated from the first day iirc, The template generated the Issue without the pipeline and I just copy pasted it over. I'm sorry for not understanding previously.

Reply:
Yes, the key is not included in the template because users always changed it to their language which makes no sense and will break search.